### DISCUSSION TITLE
Why is it that when using the flag -fe command with different regular expressions, the same results are returned?

### DISCUSSION THREAD

**Discussion by no0bsec** [projectdiscovery/httpx#1145] [Q&A] [answered]
### httpx version:

v1.3.0
### Current Behavior:

According to the meaning of -fe [FAILED], it should return results that do not contain [FAILED].But what my terminal returns is:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/45028098/236385104-b0b0cd8b-392a-4cba-a4fe-f58e26ac29e7.png">

This result is consistent with the result returned by -fe [SUCCESS].
<img width="557" alt="image" src="https://user-images.githubusercontent.com/45028098/236385439-3e3b6ed6-64f0-4c66-9aba-4990ecf46e1a.png">

If my regular expression is [SUCCESS], then the returned result conforms to the parameter description.
<img width="683" alt="image" src="https://user-images.githubusercontent.com/45028098/236386058-23fe4671-13ac-456b-8efe-cdb73525a459.png">

### Steps To Reproduce:

1. Run 'httpx ..'
2. Enter the command "httpx -title -tech-detect -content-length -random-agent -status-code -timeout 5 -threads 60 -probe -fe [failed] -silent -l 2.txt"

**Comment by ehsandeep** [comment]
@no0bsec failed is the output of `-probe` option and has nothing to do with `-fe` option.

`-fe` works to filter based on http response, you can avoid using `-probe` flag if you don't wish to see failed msg in output.

**Reply by no0bsec** [reply]
Thanks for the explanation.
So, if I want to view only the failed output, should I use "-fe [SUCCESS]" or are there other flags available?

**Reply by ehsandeep** [reply, marked as answer]
@no0bsec you can use `httpx -mdc 'failed == true'` to list host failed to probe.
